staging/publishing/rules.yaml

```diff
@@ -4,7 +4,7 @@ recursive-delete-patterns:
 - BUILD.bazel
 - "*/BUILD.bazel"
 - Gopkg.toml
-default-go-version: 1.14.6
+default-go-version: 1.15.0
```

Comment:
```suggestion
default-go-version: 1.15
```

Reply:
Would need to do this for all 1.15 versions below

Reply:
Oh geez, really?
_shakes fist at go minor versions_

Reply:
Fixed.